src/apprt/gtk/Surface.zig

```diff
@@ -1256,8 +1256,10 @@ pub fn setClipboardString(
         const clipboard = getClipboard(self.gl_area.as(gtk.Widget), clipboard_type) orelse return;
         clipboard.setText(val);
 
-        // We only toast if we are copying to the standard clipboard.
+        // We only toast if we are copying to the standard clipboard
+        // and that the new content is non-empty
```

Comment:
I think that it would be good to have "Clipboard Cleared" toast as a possibility, although I can see some people wanting to turn that off independent of the "Clipboard Copied" toast.

Reply:
Hmmmm... fair enough